sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -42,11 +49,13 @@ export function buildQueueOptions(queue: CreateQueueOptions): InternalQueueOptio
     EnableBatchedOperations: getStringOrUndefined(queue.enableBatchedOperations),
     AuthorizationRules: getRawAuthorizationRules(queue.authorizationRules),
     Status: getStringOrUndefined(queue.status),
+    ForwardTo: getStringOrUndefined(queue.forwardTo),
+    UserMetadata: getStringOrUndefined(queue.userMetadata),
     AutoDeleteOnIdle: getStringOrUndefined(queue.autoDeleteOnIdle),
     EnablePartitioning: getStringOrUndefined(queue.enablePartitioning),
     ForwardDeadLetteredMessagesTo: getStringOrUndefined(queue.forwardDeadLetteredMessagesTo),
-    ForwardTo: getStringOrUndefined(queue.forwardTo),
-    UserMetadata: getStringOrUndefined(queue.userMetadata),
+
+    // TODO: can't find this in the .net ATOM library.
```

Comment:
Not sure what the action item here is...I thought this comment was just getting removed.

Reply:
EntityAvailabilityStatus is missing in .NET.

Reply:
I'll remove the comment and log an issue in .NET repo.

Reply:
The action item would have been to log an issue in the .Net repo, but looks like I already did that months ago :) https://github.com/Azure/azure-sdk-for-net/issues/14897
cc @JoshLove-msft 

We can remove this comment
```suggestion
```